automan/edm_cluster_manager.py

```diff
@@ -0,0 +1,101 @@
+import os
+import sys
+from textwrap import dedent
+import subprocess
+import re
+
+from .cluster_manager import ClusterManager
+
+
+class EdmClusterManager(ClusterManager):  # pragma: no cover
+
+    # The path to edm root on the remote, this is a relative path
+    # and is relative to the home directory.
+    EDM_ROOT = '.edm'
+
+    BOOTSTRAP = dedent("""\
+    #!/bin/bash
+
+    set -e
+    EDM_ROOT=%s
+    ENV_FILE="{project_name}/bundled_env.json"
+    EDM_EXE=~/$EDM_ROOT/bin/edm
```

Comment:
So instead of assuming this path which is not correct on systems where edm is installed as a proper package (.deb/rpm or on Mac OS), you can do what is done here: https://github.com/pypr/automan/blob/master/automan/cluster_manager.py#L80-L84